docs/_guide/styles.md

```diff
@@ -4,66 +4,310 @@ title: Styles
+      color: ${mainColor}
+    }
   `;
 }
 ```
 
-{% include project.html folder="docs/style/hostselector" openFile="my-element.js" %}
+### Define styles in a style element {#style-element} 
+
+We recommend using static styles to style LitElement components. However, in some cases you may want to evaluate and apply styles per instance, rather than to all instances of a LitElement component. One way to do this is to include inline styles in a `<style>` element in your template, and use your element's properties in your CSS rules to evaluate styles per instance.
+
+<div class="alert alert-warning">
 
-See the MDN documentation on [:host](https://developer.mozilla.org/en-US/docs/Web/CSS/:host), [:host()](https://developer.mozilla.org/en-US/docs/Web/CSS/:host()), and [pseudo-classes](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes) for more information.
+**Expressions inside a `<style>` element won't update per instance in ShadyCSS**. Due to limitations of the ShadyCSS polyfill, you can't use element properties in CSS rules as the expressions won't be evaluated.
 
-### Set host element display styles
+</div>
+
+```js
+import {LitElement, property} from 'lit-element';
+
+class MyElement extends LitElement {
+  @property() mainColor = 'blue';
+  render() {
+    return html`
+      <style>
+        :host {
+          color: ${this.mainColor};
+        }
+      </style>
+    `;
+  }
+}
+```
+
+### Define styles in an external stylesheet {#external-stylesheet}
+
+We strongly recommend static styles, CSS custom properties, or [lit-html's `classMap` or `styleMap` directives](TODO) if you're stying non-host shadow root contents.
+
+However, you can load an external stylesheet into a shadow root with a `<link>` element:
+
+```js
+import {LitElement} from 'lit-element';
+
+class MyElement extends LitElement {
+  render() {
+    return html`
+      <link rel="stylesheet" href="./styles.css">
+    `;
+  }
+}
+```
+
+This can be a good way to load CSS generated from tools like SASS/LESS.
+
+There are some important caveats though:
+
+* External styles can cause a flash-of-unstyled-content (FOUC) while they load.
+* The URL in the `href` attribute is relative to the _main document_, making this technique mostly useful for application elements where asset URLs are well known, and not for reusable elements published publicly.
+
+### Write CSS styles for a host element and its shadow DOM {#shadow-dom-styles}
+
+In this section: 
+
+* [Write CSS styles for a host element](#host-styles). The host element is your component. It is called the "host" because it hosts shadow DOM. You can style the element itself with the special CSS selectors `:host` and `:host()`.
+* [Write CSS styles for elements in shadow DOM](#shadow-dom-styles).
+* [Write CSS styles for slotted children](#slotted-styles). Light DOM children of your host element can be rendered in shadow DOM via the `slot` element. You can style slotted content with the `::slotted()` CSS selector.
```

Comment:
Maybe leave off the second sentence, and change the first to just define "slotted children", like:

> Slotted elements are light DOM children of the host that are rendered into a shadow root with the `<slot>` element.